config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :sessions
```

Comment:
`sessions` is a near-conflict to a built in rails method called `session`. I'd suggest picking another name for this just to avoid confusion. I don't think it would actually cause a conflict in the code, but it's a bit risky.